src/main/java/games/strategy/engine/random/InternalDiceServer.java

```diff
@@ -28,11 +27,6 @@ public EditorPanel getEditor() {
   }
 
   @Override
-  public boolean sameType(final IBean other) {
-    return other.getClass() == InternalDiceServer.class;
-  }
```

Comment:
Does `InternalDiceServer` also need to override `equals()` and `hashCode()` as the other beans?